core-web/src/main/java/org/silverpeas/core/web/util/viewgenerator/html/SupportedJavaScriptPlugins.java

```diff
@@ -178,5 +178,9 @@
    * External plugin to centralize the management of lists of users and groups on client-side<br/>
    * Please take a look on displayListOfUsersAndGroups.tag
    */
-  listOfUsersAndGroups
+  listOfUsersAndGroups,
+  /**
+   * External plugin to centralize the management of Silverpeas's layout
+   */
+  layout
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)